Riot/Views/RoomTitle/RoomTitleView.xib

```diff
@@ -19,7 +19,7 @@
                     <rect key="frame" x="249" y="9" width="103" height="22"/>
                     <color key="backgroundColor" red="0.0" green="0.0" blue="0.0" alpha="0.0" colorSpace="custom" customColorSpace="sRGB"/>
                     <accessibility key="accessibilityConfiguration" identifier="DisplayNameTextField"/>
-                    <fontDescription key="fontDescription" type="system" weight="medium" pointSize="18"/>
+                    <fontDescription key="fontDescription" style="UICTFontTextStyleHeadline"/>
```

Comment:
I would prefer update `RoomTitleView` class later in another PR.
Please remove your changes for the moment.